packages/chat_repository_interface/lib/src/models/message_model.dart

```diff
@@ -15,6 +24,7 @@ class MessageModel {
     required this.imageUrl,
     required this.timestamp,
     required this.senderId,
+    this.status = MessageStatus.sending,
```

Comment:
The default value should be the standard status before this feature, so I'd say MessageStatus.sent.